src/Payment.ts

```diff
@@ -164,7 +168,8 @@ export class Payment {
     senderAddress: string,
     receiverAddress: string,
     value: number | string,
-    options: PaymentOptions = {}
+    options: PaymentOptions = {},
+    extraData = '0x'
```

Comment:
I would add to the documentation that it will return the path and the estimated fees and gas. Then it is maybe not so surprising that it takes the extraData. 
Suggestion: rename it to `getTransferPathInfo` or similar?

Reply:
I am fine renaming but I thought renaming on things that are exported in the clientlib is annoying to the mobile app?

Reply:
There is only one file where we access stuff of the clientlib so it is easy to check. https://github.com/trustlines-network/mobileapp/blob/develop/src/shared/lib/client.js
We do not use it directly.

Reply:
Fixed in other PR